### PR TITLE
[Feature] ThreatStream v2 - Anomali import without approval

### PR DESCRIPTION
**Pull request by DeanArbel**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/27002
fixes: https://github.com/demisto/etc/issues/28818

## Description
Reimplemented `threatstream-import-indicator-without-approval` command.

## Must have
- [x] Tests
- [x] Documentation 

@kirbles19 Please review the following files:
- Packs/Anomali_ThreatStream/Integrations/Anomali_ThreatStream_v2/Anomali_ThreatStream_v2.yml
- Packs/Anomali_ThreatStream/ReleaseNotes/1_0_2.md
- Packs/Anomali_ThreatStream/Integrations/Anomali_ThreatStream_v2/README.md

### CODE REVIEW

**Comment by DeanArbel** [comment]
@kirbles19 Please review the following files:
- Packs/Anomali_ThreatStream/Integrations/Anomali_ThreatStream_v2/Anomali_ThreatStream_v2.yml
- Packs/Anomali_ThreatStream/ReleaseNotes/1_0_2.md

**Comment by HemaHall** [comment]
@DeanArbel and @yaakovi - Customer is asking for a status, I see merging is blocked. Can you help me with the status?

File: Packs/Anomali_ThreatStream/Integrations/Anomali_ThreatStream_v2/Anomali_ThreatStream_v2.yml [hunk 1]
```diff
@@ -618,66 +618,84 @@ script:
       be used.'
     execution: false
     name: threatstream-import-indicator-with-approval
-    outputs:
```
Comment (yaakovi) [inline Packs/Anomali_ThreatStream/Integrations/Anomali_ThreatStream_v2/Anomali_ThreatStream_v2.yml:621]: why was this removed?

File: Packs/Anomali_ThreatStream/Integrations/Anomali_ThreatStream_v2/Anomali_ThreatStream_v2.py [hunk 2]
```diff
@@ -627,6 +630,44 @@ def import_ioc_with_approval(import_type, import_value, confidence="50", classif
...
+        source_confidence_weight=source_confidence_weight,
+        expiration_ts=expiration_ts,
+        severity=severity,
+        tags=tags,
+        trustedcircles=trustedcircles
+    )})
+
+    params = build_params()
+    res = http_request("PATCH", "v1/intelligence/", params=params, json=ioc_to_import, text_response=True)
+    return_outputs("The data was imported successfully.", {}, res)
```
Comment (yaakovi) [inline Packs/Anomali_ThreatStream/Integrations/Anomali_ThreatStream_v2/Anomali_ThreatStream_v2.py:668]: just curious, what is the response here?

**Review by yaakovi — APPROVED** [review]
update integration README as well

**Comment by yaakovi** [comment]
> @DeanArbel and @yaakovi - Customer is asking for a status, I see merging is blocked. Can you help me with the status?

should be merged in the upcoming day or two.

File: Packs/Anomali_ThreatStream/Integrations/Anomali_ThreatStream_v2/Anomali_ThreatStream_v2.yml (hunk 1 above)
Comment (DeanArbel) [reply Packs/Anomali_ThreatStream/Integrations/Anomali_ThreatStream_v2/Anomali_ThreatStream_v2.yml:621]: Because the API has no outputs.

File: Packs/Anomali_ThreatStream/Integrations/Anomali_ThreatStream_v2/Anomali_ThreatStream_v2.py (hunk 2 above)
Comment (DeanArbel) [reply Packs/Anomali_ThreatStream/Integrations/Anomali_ThreatStream_v2/Anomali_ThreatStream_v2.py:668]: In a normal response, it's expected to be empty.
image

**Comment by kirbles19** [comment]
@richardbluestone Just have to review the changes in the YML, please touch base with Dean.

**Comment by richardbluestone** [comment]
I reviewed the RN and Yaml file. You should regenerate the Readme file as I made a lot of changes to the YAML file (there were a few bugs in the  argument/context path descriptions - not the part that you rewrote).
